### DISCUSSION TITLE
Victron charges altrough "Allow to charge" is NO

### DISCUSSION THREAD

**Discussion by exenbema** [Louisvdw/dbus-serialbattery#341] [General]
hello, i have Victron System & Lifepo4 2x11kwh blocks with daly bms and its good up and running. in the last week i saw a new thing: one of the batteries said in the Details IO "Allow to charge" NO - but i see that is charging. i remember that this was ok, some weeks ago - when the battery stated this setting then victron system stopps charging. 

its not a real problem because i limit charging until 53,6V - so its safa, but im wondering...

thx in advance for any ideas 
kind regards markus

VenusOS (V2.91)

latetest Firmware serialdriver V013
Battery/BMS 
BMS/Battery type: [Daly]
Cells: [16 per block) 16x230AH
Interface: [ USB to cerbo]

**Comment by Louisvdw** [comment]
Hi @exenbema 
Victron see that setting from the battery as a suggestion. They will try and do it, but the stability of the system is more important and so most of the time that will be ignored, but only for smaller currents. 

Victron is a voltage based system. So while the battery is not at the required voltage it will allow more charge current. This would normally only be an issue if there is a problem with one of the battery settings, or if a major cell imbalance happens. 
In most cases that battery is used as a buffer. The power system ramps up/down changing from the MPPT to match the system loads, but there is a bit of a delay. The battery is used to buffer this so it might be charged/discharged to supply what is requered by the load while the MPPTs is catching up. 
If you don't have MPPT chargers you might still see it, but it should be much less.

**Reply by exenbema** [reply]
![test](https://user-images.githubusercontent.com/114774449/207070640-66899e07-02dc-427d-a74a-0a6b8598d368.png)
hi @Louisvdw that helped a lot! 

my shunt is more excat in counting, we all know that, he counts even the smaller currents as well. my daly goes faster to 100% and then lowers the charging current - what is extremly bad because i m really maybe at 70% (compared to the voltage) and the bms or the driver in the cerbo lowers the charging speed drastically - in that case around 5A. like today i had full sun, but charging was soo slow and when i watch the voltage is not high - not in the region of around 54V 

one time i set the daly bms to 100% as the batteries was full charged at 55V. but 2 weeks after that the settings are differnt as before. always the daly goes faster to SOC 90, 95, 100% .... and therefore lowers the charging current.... 

so is there a possibility that i tell the driver calculate the SOC in another way - or tell the daly bms to count smaller currents?

thx markus

**Comment by Louisvdw** [comment]
There is no way to tell the BMS how to calculate the SOC. That locig is build into the BMS firmware. But you can adjust the settings that influence that logic.

**Reply by exenbema** [reply]
thx ok i see ! maybe can you recommand a site where this is in detail explained - you are so into this topic - thx for that

**Reply by exenbema** [reply]
its solved now because the version 014 is new on voltage based - so happy:)
